### PR TITLE
BUG 1770642: Set PF MTU when deviceType is vfio

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -230,16 +230,15 @@ func setSriovNumVfs(pciAddr string, numVfs int) error {
...
-	ifaceName, err := dputils.GetNetNames(pciAddr)
-	if err != nil {
-		glog.Warningf("setNetdevMTU(): fail to get interface name for %s: %s", pciAddr, err)
-		return err
-	}
-	mtuFile := "net/" + ifaceName[0] + "/mtu"
-	mtuFilePath := filepath.Join(sysBusPciDevices, pciAddr, mtuFile)
 	b := backoff.NewConstantBackOff(1 * time.Second)
-	err = backoff.Retry(func() error {
+	err := backoff.Retry(func() error {
```
Comment (zshi-redhat) [inline pkg/utils/utils.go]: could you explain why above lines are moved into `backoff.Retry()`?

Comment (pliurh) [reply pkg/utils/utils.go]: Sometimes, after VFs are created, the sysfs is not updated immediately. In that case, the `dputils.GetNetNames()` call will failed with `file/directory not exist`. Therefore I move it into the retry cycle.

**Pull request merged**